api/src/legacy_soap_api/legacy_soap_api_routes.py

```diff
@@ -60,7 +60,10 @@ def simpler_soap_api_route(
             auth=get_soap_auth(request.headers.get(MTLS_CERT_HEADER_KEY)),
             operation_name=operation_name,
         )
-        soap_proxy_response = get_proxy_response(soap_request)
+        if skip_soap_proxy(soap_request.operation_name, soap_request.data):
```

Comment:
Putting "500" here for the status code because that's the status code for not found from the soap proxy